components/OpportunityCard.jsx

```diff
@@ -0,0 +1,146 @@
+import styled from "styled-components";
+import { Text } from "./Text";
+
+export const OpportunityCard = ({
+  name,
+  location,
+  category,
+  organization,
+  postingDate,
+  logo,
+	link,
+  ...props
+}) => {
+  return (
+    <Wrapper {...props}>
+      <Logo src={logo} />
+      <InfoContainer>
+        <Name href={link}>{name}</Name>
+        <Organization>{organization}</Organization>
+        <Category>{category}</Category>
+        <BottomText>
+          <Location>{location}</Location>
+          <PostingDate>Posted {postingDate}</PostingDate>
+        </BottomText>
+      </InfoContainer>
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.div`
+  ${({ theme }) => `
+		border-color: ${theme.font.text};
+		box-shadow: ${theme.boxShadow.topBottom};
+	`};
+  border-radius: 20px;
+  display: flex;
+  flex-direction: row;
+  height: 130px;
+  padding: 10px;
+  margin: 10px;
+  align-items: center;
+  @media only screen and (max-width: 900px) {
+    height: 100px;
+    padding: 5px;
```

Comment:
Is there a reason for not using the media util? 
```
${media(
"breakpoint", 
`styles`
)};
```

Reply:
media util doesn't interact well with getting stuff from props or `theme` - it's been done this way in other parts of the codebase for similar reasons

Reply:
I don't think there's an issue with media util if you're not using the theme/props inside of it (but I know other media queries here do) 
```
${media (
  "tablet", 
  `
    height: 100px; 
    padding: 5px;
  `
)}; 
``` 
^ so that should be fine here

Reply:
yeah that's what I meant, I copied the `$media` statement for all of the styles so I'll change everything else over to use the media util if it doesn't need to use the theme or props